UIMenu.cs

```diff
@@ -146,6 +145,12 @@ public class UIMenu
         public bool MouseControlsEnabled = true;
         public bool ScaleWithSafezone = true;
 
+        public Point Offset;
```

Comment:
This should probably be a get-only auto property since (at this time) modifying it after menu creation won't actually have the desired effect.
```suggestion
        public Point Offset { get; }
```